.rubocop.yml

```diff
@@ -45,6 +45,9 @@ Layout/LineLength:
   IgnoredPatterns:
     - '^#*'
 
+Metrics/MethodLength:
```

Comment:
I found this rule too restrictive, I can understand rules about complexity and such but there is no universal formula about how long method should be. Sometimes dividing method into smaller methods can make reading more difficult its all relative.

Reply:
> there is no universal formula about how long method should be.

True, but I tend to favor smaller methods over larger ones. Instead of completely disabling this rule globally what do you think about disabling the rule for the code that you want to opt-out of?

E.g.

```ruby
# rubocop:disable Metrics/MethodLength
def long_method
end
# rubocop:enable Metrics/MethodLength
```

https://docs.rubocop.org/en/latest/configuration/#disabling-cops-within-source-code

Reply:
@mokhan 

I am okay with that